patches/coreboot-4.8.1/0030-sandybridge.patch

```diff
@@ -0,0 +1,58 @@
+diff --git ./src/northbridge/intel/sandybridge/romstage.c ./src/northbridge/intel/sandybridge/romstage.c
+index 0426b83..d348b9e 100644
+--- ./src/northbridge/intel/sandybridge/romstage.c
++++ ./src/northbridge/intel/sandybridge/romstage.c
+@@ -29,6 +29,8 @@
+ #include <device/device.h>
+ #include <halt.h>
+ #include <security/tpm/tis.h>
++#include <security/tpm/tss.h>
++#include <program_loading.h>
+ #include <northbridge/intel/sandybridge/chip.h>
+ #include "southbridge/intel/bd82x6x/pch.h"
+ #include <southbridge/intel/common/gpio.h>
+@@ -72,6 +74,19 @@ void mainboard_romstage_entry(unsigned long bist)
+ 	/* Initialize superio */
+ 	mainboard_config_superio();
+ 
++	if (IS_ENABLED(CONFIG_MEASURED_BOOT) && IS_ENABLED(CONFIG_LPC_TPM)) {
```

Comment:
Do we need to check for CONFIG_LPC_TPM ? The Kconfig option for CONFIG_MEASURED_BOOT already depends on MAINBOARD_HAS_LPC_TPM and it does a 'select TPM' so I assume that if MEASURED_BOOT is enabled then LPC_TPM will also always be enabled.

Reply:
Same comment as on the fsp2.0 side, seems safer in the future with the added guard.